resources/images.yaml

```diff
@@ -4,10 +4,10 @@ metadata:
   name: extra-openebs-images
 spec:
   images:
-  - image: quay.io/openebs/jiva:1.5.0
-  - image: quay.io/openebs/cstor-istgt:1.5.0
-  - image: quay.io/openebs/cstor-pool:1.5.0
-  - image: quay.io/openebs/cstor-pool-mgmt:1.5.0
-  - image: quay.io/openebs/cstor-volume-mgmt:1.5.0
-  - image: quay.io/openebs/m-exporter:1.5.0
-  - image: quay.io/openebs/linux-utils:1.5.0
+  - image: quay.io/openebs/jiva:1.7.0
```

Comment:
Is there any way this could be automated or minimized to changing a version in the Makefile?

Reply:
Yeah, we have the `--set-image` flag which is actually already used when building storage app so I don't know why I keep updating these versions manually as well :) I'll see if I can just replace all image versions with `0.0.0` and have tele replace them.